config/core/test-configs.yaml

```diff
@@ -2245,6 +2253,8 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-lkdtm
+      - kselftest-seccomp
```

Comment:
Looks good to me, this is a new device type with 5 instances online so running several kselftest plans should be fine.